app/commands.py

```diff
@@ -804,3 +804,32 @@ def populate_service_volume_intentions(file_name):
             service.volume_letter = columns[3]
             dao_update_service(service)
     print("populate-service-volume-intentions complete")
+
+
+@notify_command(name='populate-go-live')
+@click.option('-f', '--file_name', required=True, help='CSV file containing live service data')
+def populate_go_live(file_name):
+    # 0 - count, 1- Link, 2- Service ID, 3- DEPT, 4- Service Name, 5- Main contact,
+    # 6- Contact detail, 7-MOU, 8- LIVE date, 9- SMS, 10 - Email, 11 - Letters, 12 -CRM, 13 - Blue badge
+
+    print("Populate go live user and date")
+    with open(file_name, 'r') as f:
+        for line in itertools.islice(f, 1, None):
```

Comment:
Would be better to use the Python’s CSV parsing code here, to deal with stuff like cells containing commas. See how it’s used here: https://github.com/alphagov/notifications-utils/blob/master/notifications_utils/recipients.py#L146-L150

Reply:
Ok - done